src/libiotc/iotc_user_sub_call_wrapper.c

```diff
@@ -73,8 +73,12 @@ iotc_state_t iotc_user_sub_call_wrapper(void* context, void* data,
           msg->publish.content ? msg->publish.content->data_ptr : NULL;
       params.message.temporary_payload_data_length =
           msg->publish.content ? msg->publish.content->length : 0;
-      params.message.topic = (const char*)msg->publish.topic_name->data_ptr;
 
+      // issues/111: Make sure we null terminate the string.
```

Comment:
Not really sure if `// issues/111` is the right way to annotate a line to get a github link.
@atigyi could you clarify? Thanks

Reply:
#111